app/src/main/java/com/dessalines/thumbkey/keyboards/ENMessageEaseSymbols.kt

```diff
@@ -892,12 +892,12 @@ val KB_EN_MESSAGEEASE_SYMBOLS_SHIFTED =
 
 val KB_EN_MESSAGEEASE_SYMBOLS: KeyboardDefinition =
     KeyboardDefinition(
-        title = "english symbols messageease",
+        title = "english symbols messagease",
```

Comment:
Ah, I had that wrong. Might as well do a search / replace on all these then.

Reply:
I'll sort this out later. I was wondering about the nuisance of doing all the symbols too... For the titles, are you happy to capitalise MessagEase properly too?